documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -362,6 +365,37 @@ ifdef::community[]
 == Confluent Schema Registry
 
 There is an alternative https://github.com/confluentinc/schema-registry[schema registry] implementation provided by Confluent.
+
+[id="overview-of-deploying-confluent-schema-registry"]
+=== Confluent Schema Registry deployment overview
+
+To install standalone Confluent Schema Registry, please see Confluent's https://docs.confluent.io/platform/current/installation/installing_cp/overview.html[official deployment guide].
+
+[NOTE]
+====
+There's also possibility to https://docs.confluent.io/platform/current/installation/docker/installation.html[install] it as a container.
+====
+
+[id="deploying-confluent-schema-registry-with-debezium-containers"]
+=== Deploying Confluent Schema Registry with {prodname} containers
+
+As of {prodname} 2.0.0, Confluent Schema Registry support is on part of the {prodname} containers.
+To make it working, you need to install into Connect plugin directory appropriate jar files which contain Confluent Avro converters.
+Namely, you would need following jar file:
+
+* `kafka-connect-avro-converter`
+* `kafka-connect-avro-data`
+* `kafka-avro-serializer`
+* `kafka-schema-serializer`
+* `kafka-schema-registry-client`
+* `common-config`
+* `common-utils`
+
+[NOTE]
+====
+You can download the required jar files https://packages.confluent.io/maven/[Confluent's Maven repository].
+====
```

Comment:
```suggestion
You can download the preceding files from the https://packages.confluent.io/maven/[Confluent Maven repository].
```
Similar to my earlier comment, you can drop the note formatting here and just add this information in-line as a standard paragraph.